src/caret_analyze/record/frequency.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Tuple
+
+from .column import ColumnValue
+from .interface import RecordsInterface
+from .record_factory import RecordFactory, RecordsFactory
+
+
+class Frequency:
+
+    def __init__(
+        self,
+        records: RecordsInterface,
+        target_column: Optional[str] = None
+    ) -> None:
+        self._target_column = target_column or records.columns[0]
+        self._target_series = [ts for ts in records.get_column_series(
+                               self._target_column) if ts is not None]
+
+    def to_records(
+        self,
+        interval_ns: int = 1000000000,
+        base_timestamp: Optional[int] = None
+    ) -> RecordsInterface:
```

Comment:
Could you add docstring to describe arguments?

Reply:
added: https://github.com/tier4/CARET_analyze/pull/116/commits/78088a4f51dcbe85ba3efda35e2c988c2e14eeb9